### PR TITLE
DM-13520 Create readme.md

### PR DESCRIPTION
**Pull request by parejkoj**
Create a readme and add a note about the HSC focalplane image.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# obs_subaru
+
+Subaru-specific (hsc,  hscSim, suprimecam) configuration and tasks for the LSST Data Management Stack
```
Comment (PaulPrice) [inline README.md]: `hscSim` is long dead.

Comment (parejkoj) [reply README.md]: I lifted that description from the package description on the main page... I'll fix it here, but someone (admin?) needs to fix it on the main github page.

**Comment by PaulPrice** [comment]
Yeah, I don't have r00t. @timj?

**Comment by timj** [comment]
@PaulPrice what are you asking me to do?

**Comment by ktlim** [comment]
I did it.

**Pull request merged**